test/models/cart_test.rb

```diff
@@ -0,0 +1,17 @@
+class CartTest < ActiveSupport::TestCase
+  test 'cart has initial contents' do
+    cart = Cart.new({ "1" => 1})
+
+    assert_equal({ "1" => 1}, cart.contents)
+  end
+
+  test 'can add an item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.